servers/visual_server.cpp

```diff
@@ -283,7 +283,7 @@ RID VisualServer::make_sphere_mesh(int p_lats, int p_lons, float p_radius) {
 	return mesh;
 }
 
-RID VisualServer::material_2d_get(bool p_shaded, bool p_transparent, bool p_cut_alpha, bool p_opaque_prepass) {
+RID VisualServer::material_2d_get(bool p_shaded, bool p_transparent, bool p_cut_alpha, bool p_opaque_prepass, bool p_double_sided) {
```

Comment:
Done in https://github.com/godotengine/godot/pull/8434/commits/92d9cc53d3e4d7b625cf1d35fcf7feac3c9d39bd. Can rebase if desired.